src/ia-account-settings.ts

```diff
@@ -117,75 +116,81 @@ export class IAAccountSettings
    * open delete form
    *
    * @private
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAUXAccountSettings
    */
-  @state() private attemptToDelete?: Boolean;
+  @state() private attemptToDelete?: boolean;
 
   /**
    * enable delete button when you make sure
    *
    * @private
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAUXAccountSettings
    */
-  @state() private confirmDelete?: Boolean = false;
+  @state() private confirmDelete: boolean = false;
 
   /**
    * object that contains updated fields data/text
    *
    * @private
-   * @type {Object}
+   * @type {ResponseModel}
    * @memberof IAAccountSettings
    */
-  @state() private responseFields?: ResponseModel = {};
+  @state() private responseFields: ResponseModel = {};
 
   /**
    * determine if need to disable save button
    *
-   * @private
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAAccountSettings
    */
-  @state() saveButtonDisabled?: Boolean = true;
+  @state() saveButtonDisabled: boolean = true;
 
   /**
    * determine if want to show authenticate page
    *
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAUXAccountSettings
    */
-  @state() lookingToAuth?: Boolean = true;
+  @state() lookingToAuth: boolean = true;
 
   /**
    * determine if need to show loading indicator on buttons
    *
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAUXAccountSettings
    */
-  @state() showLoadingIndicator?: Boolean;
+  @state() showLoadingIndicator?: boolean;
 
   /**
    * set true if want to make header sticky
    *
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAUXAccountSettings
    */
-  @state() isStickyHeader?: Boolean = false;
+  @state() isStickyHeader: boolean = false;
 
   /**
    * toggle password field type to text|password
-   * @type {Boolean}
+   * @type {boolean}
    * @memberof IAUXAccountSettings
    */
-  @state() private showPassword?: Boolean = false;
+  @state() private showPassword: boolean = false;
 
   /**
    * flag to verify if API has executed
-   * @type {Boolean}
+   * @type {boolean}
+   * @memberof IAUXAccountSettings
+   */
+  @state() private apiHasExecuted: boolean = false;
+
+  /**
+   * Stores the authentication method used (e.g. 'ia' for Internet Archive)
+   * @type {string}
    * @memberof IAUXAccountSettings
```

Comment:
Also, I know we've got all these `@type` and `@memberof` typedocs in here already so no need to change any of that, but since typescript already contains all of that type information as part of the code (ie `apiHasExecuted: boolean`), those typedocs are a bit more redundant and adds one more spot to remember to update if anything changes. Those are more important for JS where we don't have type info. The description is super useful, though!